src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/tests/latest/test_acr_commands.py

```diff
@@ -271,7 +271,7 @@ def test_acr_create_replication(self, resource_group, resource_group_location):
     @ResourceGroupPreparer()
     @record_only()
     def test_acr_image_import(self, resource_group):
-        '''There are nine test cases in the function.
+        '''There are nine test cases.
```

Comment:
There are much more cases, please cover every combination of parameters, including the command that triggered the issue.

Reply:
Got it. The cross subscription image import scenario.

Reply:
Not just that, list all combinations.